core/workflow-operator/src/main/scala/edu/uci/ics/amber/operator/metadata/OperatorMetadataGenerator.scala

```diff
@@ -156,4 +156,20 @@ object OperatorMetadataGenerator {
     )
   }
 
+  /**
+    * Explicitly start a thread to let the objectMapper load all logical operator classes for serde/deserde.
+    *
+    * This call prevents the initial delay of serialization & deserialization in other application logics.
+    */
+  def warmupObjectMapperForOperatorsSerde(): Unit = {
+    val thread = new Thread(
+      new Runnable {
+        override def run(): Unit = {
+          OperatorMetadataGenerator.generateAllOperatorMetadata()
+        }
+      },
+      "ObjectMapperWarmupForOperatorsThread"
+    )
+    thread.start()
+  }
```

Comment:
define this warmupObjectMapperForOperatorsSerde() function to edu.uci.ics.amber.util.JSONUtils. it does not belong to OperatorMetadataGenerator

Reply:
JSONUtils is in `workflow-core`, it cannot see `OperatorMetadataGenerator `

Reply:
I moved it to a new object called `ObjectMapperUtils`